### PR TITLE
fix(ext/node): make worker ids sequential

### PR DESCRIPTION
**Pull request by satyarohith**
Requires https://github.com/denoland/deno/pull/22647 for the tests to work.

Closes https://github.com/denoland/deno/issues/22812

### CODE REVIEW

File: runtime/web_worker.rs [hunk 1]
```diff
@@ -630,11 +637,13 @@ impl WebWorker {
         v8::String::new(scope, &format!("{}", self.id))
           .unwrap()
           .into();
```
Comment (bartlomieju) [inline runtime/web_worker.rs:639]: Won't this cause a change of name (because we have a wrapper struct)?

**Review by bartlomieju — APPROVED** [review]
LGTM, one question that would be worth to be verified before landing

File: runtime/web_worker.rs (hunk 1 above)
Comment (satyarohith) [reply runtime/web_worker.rs:639]: The `id_str` value will still be `worker-{id}` due to `fmt::Display` implementation or did I not understand your question?

Comment (bartlomieju) [reply runtime/web_worker.rs:639]: Ah, you're right

**Pull request merged**